tests/build/multi-platform.go

```diff
@@ -304,6 +449,165 @@ func getAwsInstances() ([]string, error) {
 	return ret, nil
 }
 
+func createConfigMapForHostPool(f *framework.Framework) error {
+	armInstances, err := getHostPoolAwsInstances()
+	if err != nil {
+		return fmt.Errorf("error getting aws host pool instances: %v", err)
+	}
+	hostConfig := &v1.ConfigMap{}
+	hostConfig.Name = HostConfig
+	hostConfig.Namespace = ControllerNamespace
+	hostConfig.Labels = map[string]string{MultiPlatformConfigKey: "hosts"}
+
+	hostConfig.Data = map[string]string{}
+	count := 0
+	for _, instance := range armInstances {
+		hostConfig.Data[fmt.Sprintf("host.aws-arm64-%d.address", count)] = instance
+		hostConfig.Data[fmt.Sprintf("host.aws-arm64-%d.platform", count)] = AwsPlatform
+		hostConfig.Data[fmt.Sprintf("host.aws-arm64-%d.user", count)] = Ec2User
+		hostConfig.Data[fmt.Sprintf("host.aws-arm64-%d.secret", count)] = AwsSecretName
+		hostConfig.Data[fmt.Sprintf("host.aws-arm64-%d.concurrency", count)] = "4"
+		count++
+	}
+
+	_, err = f.AsKubeAdmin.CommonController.CreateConfigMap(hostConfig, ControllerNamespace)
+	if err != nil {
+		return fmt.Errorf("error creating host-pool config map: %v", err)
+	}
+	return nil
+}
+
+func createConfigMapForDynamicInstance(f *framework.Framework) error {
+	hostConfig := &v1.ConfigMap{}
+	hostConfig.Name = HostConfig
+	hostConfig.Namespace = ControllerNamespace
+	hostConfig.Labels = map[string]string{MultiPlatformConfigKey: "hosts"}
+
+	hostConfig.Data = map[string]string{}
+	hostConfig.Data["dynamic-platforms"] = AwsPlatform
+	hostConfig.Data["instance-tag"] = DynamicInstanceTag
+	hostConfig.Data["dynamic.linux-arm64.type"] = "aws"
+	hostConfig.Data["dynamic.linux-arm64.region"] = AwsRegion
+	hostConfig.Data["dynamic.linux-arm64.ami"] = "ami-09d5d0912f52f9514"
+	hostConfig.Data["dynamic.linux-arm64.instance-type"] = "t4g.medium"
```

Comment:
Can we get away with a cheaper instance for the tests?

Reply:
Good point. Used `t4g.micro` now.

When tried `t4g.nano`, getting below error
```
2024-03-12T07:27:01Z    ERROR   taskrun taskrun/dynamic.go:120  Failed to create cloud host     {"request": {"name":"multi-platform-suite-component-qntn-6887z-build-container","namespace":"multi-platform-d-2-tenant"}, "error": "operation error EC2: RunInstances, https response error StatusCode: 400, RequestID: d4bc9ddb-77d3-4b80-b13f-cf792af8cbe9, api error UnsupportedOperation: Red Hat Enterprise Linux is not supported for the instance type 't4g.nano'."}
```